src/main/java/creativei/controller/MainController.java

```diff
@@ -17,9 +14,24 @@
 
     @Autowired
     BranchManager branchManager;
+    @Autowired
+    InquiryManager inquiryManager;
 
     @RequestMapping("/")
     public String index() {
         return "Greetings from Spring Boot!";
     }
+
+    @RequestMapping(value="/pincode/beginWith",produces ="application/json",method = RequestMethod.GET)
+    public @ResponseBody
+    ResponseObject getPincodeStartWith( @RequestParam String pincode){
+      ResponseObject responseObject=  inquiryManager.getPincodes(pincode);
```

Comment:
Why are we using inquiry manage for pincodes? Use proper abstraction hirarchy

Reply:
Should i make a new Manager Layer for this

Reply:
Yes

Reply:
Added Locality manager.PR updated in commit 5f8c87a